### ISSUE TITLE
Verify that all relevant sniffs handle multi-property declarations correctly

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#1722]
Several class properties can be declared at once:
```php
class ABC {
    public $first, $second, $third;

    protected
        $withValueA = true,
        $withValueB = self::SOME_CONSTANT,
        $withValueA = 4 + 3;
}
```

PRs #1717, #1720 and #1721 improve the `File::getMemberProperties()` method and two sniffs to handle this type of syntax.

However, there might be more sniffs which need reviewing on whether they can handle this correctly.

**Comment by gsherwood** [comment]
Those PRs have been merged. I'm going to close this in favour of new bug reports if any issues are found.
